src/html/partials/header.html

```diff
@@ -1,8 +1,18 @@
 <header>
+
   <div class="wrapper--title">
     <h1 class="header__title">A Viget Exploration</h1>
   </div>
-  <div>
-    <img src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
-  </div>
+
+  <img class="header__img" src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
```

Comment:
If you have `header__img` here, might want to have `header` above, e.g. `<header class="header">`, but you can always address that when you get to the css/styling phase.